examples/hello_world_datahub_cloud.yaml

```diff
@@ -0,0 +1,12 @@
+# hello_world.yaml
+name: "hello_world_datahub_cloud"
+# 1. DataHub Cloud Connection: Configure how to talk to DataHub Cloud
+datahub:
+  server: "<your-datahub-cloud-url"
+  token: "<your-datahub-cloud-token>"
+# 2. Event Source: Where to source event from.
+source:
+  type: "datahub-cloud"
+# 3. Action: What action to take on events. 
+action:
+  type: "hello_world"
```

Comment:
@jjoyce0510 could we add an example of how to reference a specific action here? Or an example action? I still wouldn't know after reading this doc what my action might look like.

Reply:
```suggestion
  # to see how this action was developed, see https://datahubproject.io/docs/actions/guides/developing-an-action
  type: "hello_world"
```

Reply:
nice suggestion - looks good to me!